src/scss/components/_media-element.scss

```diff
@@ -0,0 +1,134 @@
+/**
+ * Media element
+ * ===================================
+ * Example – SERP – Restaurant logos
+ *
+ * A common pattern that sees an image followed by a block of information.
+ *
+ */
+
+$mediaElement-img-width: 50px !default;
+$mediaElement-img-height: 50px !default;
```

Comment:
Do you need this variable and the other height variants? They don't seem to be used anywhere here.

Reply:
@kevinrodrigues good point :)

I'll remove these, maybe they were added as an indicator of 'square logos'